internal/client/cluster/cluster_resource.go

```diff
@@ -1,18 +1,25 @@
 /*
-Copyright © 2021 VMware, Inc. All Rights Reserved.
+Copyright © 2022 VMware, Inc. All Rights Reserved.
```

Comment:
Do not change the copyrights, 
copyrights should only be added during the new file creation and not be update everytime the file gets modified

Reply:
Sure, I will revert this change.

Reply:
Updated @shreyassreenivas